docs/build-from-scratch.md

```diff
@@ -93,11 +93,10 @@ When running Heptio Ark, you will need to account for the following (all of whic
 * Cloud provider credentials
   * Read/write access to volumes
   * Read/write access to object storage for backup data
-* A [Config object][8] definition for the Ark server
```

Comment:
Let's change this line to say "A BackupStorageLocation object definition for the Ark server"
And add another line: "(Optional)" a VolumeSnapshotLocation object definition for the Ark server, if you want to be able to take PV snapshots"